src/Twig/MediaRuntime.php

```diff
@@ -150,7 +152,12 @@ private function getMedia($media): ?MediaInterface
             return null;
         }
 
-        if (MediaInterface::STATUS_OK !== $media->getProviderStatus()) {
+        $fileName = $media->getName();
+
+        if (
+            MediaInterface::STATUS_OK !== $media->getProviderStatus() &&
+            null !== $fileName && \in_array(strtolower(pathinfo($fileName, \PATHINFO_EXTENSION)), self::EXCLUDED_FORMATS, true)
```

Comment:
Prior to the `!== null` change, the check you added was 
```
!\in_array(strtolower(pathinfo($fileName, \PATHINFO_EXTENSION)), self::EXCLUDED_FORMATS, true)
```
so it should be 
```
MediaInterface::STATUS_OK !== $media->getProviderStatus() &&
            (null === $fileName || !\in_array(strtolower(pathinfo($fileName, \PATHINFO_EXTENSION)), self::EXCLUDED_FORMATS, true))
```

Reply:
made it